ndmg/stats/qa_reg.py

```diff
@@ -43,8 +43,21 @@ def reg_mri_pngs(
     mri, atlas, outdir, loc=0, mean=False, minthr=2, maxthr=95, edge=False
 ):
     """
-    outdir: directory where output png file is saved
-    fname: name of output file WITHOUT FULL PATH. Path provided in outdir.
+    A function to create and save registered brain slice figures.
+    
+    Parameter
+    ---------
+    mri: nifti file
+        the registered brain file generated in each registration step.
+    atlas: nifti file
+        the reference brain file used in each registration step.
+    outdir: str
+        directory where output png file is saved. 
+    loc: int
+    mean: bool
+    minthr: int
+    maxthr: int
+    edge: bool
```

Comment:
Include descriptions for these inputs

Reply:
see https://github.com/neurodata/ndmg/pull/372/commits/3be6eafbddb55f52d9bb5ed4b5df981b975ac532